folly/experimental/symbolizer/Dwarf.cpp

```diff
@@ -19,7 +19,11 @@
 
 #include <type_traits>
 
-#include <dwarf.h>
+#if FOLLY_HAVE_LIBDWARF_DWARF_H
+# include <libdwarf/dwarf.h>
+#elif FOLLY_HAVE_DWARF_H
```

Comment:
Make this just `#else`, otherwise it will silently not include `dwarf.h` at all. It's much clearer what an issue is when it says it can't find the header rather than not finding the symbols.